src/python/pants/option/global_options.py

```diff
@@ -46,10 +47,23 @@ class GlobMatchErrorBehavior(Enum):
 class FileNotFoundBehavior(Enum):
     """What to do when globs do not match in BUILD files."""
 
+    ignore = "ignore"
     warn = "warn"
     error = "error"
 
     def to_glob_match_error_behavior(self) -> GlobMatchErrorBehavior:
+        deprecated_conditional(
+            lambda: self == type(self).ignore,
```

Comment:
Alternatively, you can use `self.value == "ignore"`.

Reply:
To me it feels less clear what that's referencing. I'm not sure I could jump to the definition in an IDE, for example. That's probably fine for here.